chasm/src/testData/java/empty/EmptyOuterClass.java

```diff
@@ -1,5 +1,6 @@
 package empty;
 
+@SuppressWarnings("ALL")
```

Comment:
That is a lot of suppressed warnings

Reply:
it's an empty class and there are a lot of warnings so i got lazy and suppressed everything

Reply:
it's an empty class which has ASM being done on it (code which will never be executed) so i just suppressed everything

Reply:
Ah makes sense, but wouldn't suppressing "unused" be enough ?

Reply:
yeah, probably. i'll change it